src/allmydata/test/test_storage_http.py

```diff
@@ -501,13 +525,25 @@ def result_of_with_flush(self, d):
 
         # OK, no result yet, probably async HTTP endpoint handler, so advance
         # time, flush treq, and try again:
-        for i in range(100):
+        for i in range(10_000):
```

Comment:
That's ... a lot? :/

Reply:
Notice that in the most common, no-actual-async path, it'll only have one iteration regardless, it won't actually loop.

So this is just future compatibility for any future tests that take a while to run.